opentelemetry-appender-log/src/lib.rs

```diff
@@ -69,3 +69,98 @@ const fn severity_of_level(level: Level) -> Severity {
         Level::Trace => Severity::Trace,
     }
 }
+
+#[cfg(all(test, feature = "testing", feature = "logs"))]
+mod tests {
+    use super::OpenTelemetryLogBridge;
+
+    use opentelemetry_sdk::{logs::LoggerProvider, testing::logs::InMemoryLogsExporter};
+
+    use log::Log;
+
+    #[test]
+    fn logbridge_with_default_metadata_is_enabled() {
+        let exporter = InMemoryLogsExporter::default();
+
+        let logger_provider = LoggerProvider::builder()
+            .with_simple_exporter(exporter)
+            .build();
+
+        let otel_log_appender = OpenTelemetryLogBridge::new(&logger_provider);
+
+        // As a result of using `with_simple_exporter` while building the logger provider,
+        // the processor used is a `SimpleLogProcessor` which has an implementation of `event_enabled`
+        // that always returns true.
+        #[cfg(feature = "logs_level_enabled")]
+        assert_eq!(
+            otel_log_appender.enabled(&log::Metadata::builder().build()),
+            true
+        );
+        #[cfg(not(feature = "logs_level_enabled"))]
+        assert_eq!(
+            otel_log_appender.enabled(&log::Metadata::builder().build()),
+            true
+        );
+    }
+
+    #[test]
+    fn logbridge_with_record_can_log() {
+        let exporter = InMemoryLogsExporter::default();
+
+        let logger_provider = LoggerProvider::builder()
+            .with_simple_exporter(exporter.clone())
+            .build();
+
+        let otel_log_appender = OpenTelemetryLogBridge::new(&logger_provider);
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-appender-tracing/src/layer.rs#L238-L239

Can we use macros from `log` to emit the logs instead, as that is what end users would be doing...

Reply:
Good suggestion, I updated the tests to use log::* macros to emit logs